templates/cmake/openharmony.cmake

```diff
@@ -11,6 +11,13 @@ macro(cc_openharmony_after_target target_name)
     target_link_libraries(${target_name}
         ${ENGINE_NAME}
     )
+
+    target_link_options(${target_name} PRIVATE -Wl,--gc-sections)
+
+    if("${CMAKE_BUILD_TYPE}" STREQUAL "Release")
+        target_link_options(${target_name} PRIVATE -Wl,-s)
```

Comment:
Strip so in release build. `.so` file size now:

-rw-r--r--@ 1 james  staff  79434848 Jan  1  2009 libcocos-napi-debug.so
-rw-r--r--@ 1 james  staff  13582208 Jan  1  2009 libcocos-napi-release.so
-rw-r--r--@ 1 james  staff  99104128 Jan  1  2009 libcocos-v8-debug.so
-rw-r--r--@ 1 james  staff  26263536 Jan  1  2009 libcocos-v8-release.so